docs/source/requirements-sections.adoc

```diff
@@ -0,0 +1,5 @@
+// Do not edit this file directly.
+
+include::/workspaces/dawe-rlp-spec/docs/source/requirements-by-module/plot-description/slope/index.adoc[]
+
+include::/workspaces/dawe-rlp-spec/docs/source/requirements-by-module/plot-description/slope-type/index.adoc[]
```

Comment:
I need to fix this by using relative paths instead of absolute paths from the devcontainer.